guides/README.md

```diff
@@ -39,6 +39,10 @@ Similarly, to build and open PDF version do:
 
     make open-pdf
 
+To speed up the build process, make sure to use `-j` option. Ideally, set it to amount of cores plus one:
+
+		make -j9
```

Comment:
Yeah, a bit less readable so let me create two examples.

Reply:
That was what I had in mind. I also wonder if `MAKEOPTS` is a Gentoo thing so I started to search. https://stackoverflow.com/questions/4778389/automatically-setting-jobs-j-flag-for-a-multicore-machine suggests you can use `MAKEFLAGS+=j2` in your `Makefile`. Just something to consider. I'm also fine with documenting it.

 Otherwise you can recommend `MAKEOPTS="-j$(nproc)"